test/e2e/project_settings_test.go

```diff
@@ -59,19 +60,29 @@ func TestProjectSettings(t *testing.T) {
 	})
 
 	t.Run("Update", func(t *testing.T) {
-		cmd := exec.Command(cliPath,
-			projectsEntity,
-			settingsEntity,
-			"update",
-			"--disableCollectDatabaseSpecificsStatistics",
-			"--projectId",
-			g.projectID,
-			"-o=json")
-		cmd.Env = os.Environ()
-		resp, err := RunAndGetStdOut(cmd)
-		require.NoError(t, err, string(resp))
 		var settings atlasv2.GroupSettings
-		require.NoError(t, json.Unmarshal(resp, &settings))
+
+		for i := 0; i < 10; i++ { // try again for 10 seconds
```

Comment:
[q] would it be better to add `--watch` to update?

Reply:
there is no `--watch` for the api, I think there is a cache on the backend

Reply:
Ok, that's clear.

Are we sure 10 seconds is enough?

Reply:
On my local testing it took 2 seconds or less